xml/admin_install_salt.xml

```diff
@@ -1482,8 +1482,7 @@ default_drive_group_name:
 &prompt.smaster;salt-run disks.list
 </screen>
      <para>
-      This runner returns you a structure of matching disks based on your drive
-      groups. If you are not happy with the result, repeat the previous step.
+      This runner returns you a structure of matching disks based on your &drvgrps;. If you are not happy with the result, repeat the previous step.
```

Comment:
Wayyyy too long - why'd you change the wrap text?

Reply:
by mistake :-)